config.ru

```diff
@@ -26,6 +26,7 @@ app = PactBroker::App.new do | config |
   config.webhook_host_whitelist = [/.*/, "10.0.0.0/8"]
   config.webhook_scheme_whitelist = ['http', 'https']
   config.webhook_http_method_whitelist = ['GET', 'POST']
+  config.webhook_http_code_whitelist = [200, 201, 202]
```

Comment:
is it 200, 201, 202 sufficient? Or shall I also add others (e.g. 203, 204, 205, 206)?. 
I do not recommend to add 301, 302 as default ones, using the configuration people can maintain their own http code whitelist anyway.

Reply:
I reckon 204 (success, no body returned) would definitely be a success. May as well add in the other ones too, to keep the behaviour backwards compatible.

Reply:
203-206 added